go1.13_compat_test.go

```diff
@@ -0,0 +1,18 @@
+// +build go1.13
+
+package errors_test
+
+import (
+	stdlib_errors "errors"
+	"testing"
+
+	"github.com/pkg/errors"
+)
+
+func TestErrorChainCompat(t *testing.T) {
+	err := stdlib_errors.New("error that gets wrapped")
+	wrapped := errors.Wrap(err, "wrapped up")
+	if !stdlib_errors.Is(wrapped, err) {
```

Comment:
```suggestion
if !errors.Is(wrapped, err) {
```

Reply:
I think that is not necessary use the package `_test` in this case because you can use directly the std library and check against the package itself

Reply:
I wanted it to be a black box test, and that is why I put it in a separate package.

Reply:
Ok so is a good argument to me, we need to merge this #212 before that we can merge your PR, because in otherwise the travis ci will never pass. But it's look great, and I woul like to merge this ASAP :)

Reply:
Once #212 is merged, I will rebase and squash this branch.

Reply:
I do not see the utility here in actually pushing it into a separate package. You can still do blackbox testing even if _technically_ you could look inside the box.

The design is in the design, not in the arbitrarily forcing a semantic. Consider as well all the people forking the project, and now that test is going to pull _this_ package for their tests, and not their own forked repo.

Reply:
I see your point and is very reasonable, but the package itself can't be tested normally from a fork, because have the repository path hardcode in some tests, for example:

https://github.com/pkg/errors/blob/master/format_test.go#L29

So if anyone fork this repository, he/she must be moved under $GOPATH/src/github.com/pkg/errors or they also could use docker instead. Otherwise the test will never pass on local.

Furthermore this can also be resolved using go modules, but yes for now the package haven't this functionality.

But, I agree that in this case one way on another to do this black box test is similar to the other. So maybe would be ok, if we try to avoid more accoplated code with the package itself.

What do you think @puellanivis?

Reply:
Hm… fun test. But a critical difference is that the `format_test` would break loudly, while this would not even break, because a forking author may not even realize that this import is here, and thus get a false sense of security that their tests are passing, even though they are not even running against their own code.

Reply:
Yes, I totally agree that are two different case, this case could produce as you said false sense of security. My point was that with the current state of the package tests, the package couldn't be tested from another directory. But, I see your point.

@jayschwa could you change the test and use the package errors instead the package errors_test? Is truth that Go provide with the package _test to realize the black box test on secure way, but we can mantain this test as black box even without that.

Thanks 😁

PS. the PR #212 is already merged, so your pipeline should be pass this time

Reply:
I moved the test back into the `errors` package and CI is green.

Reply:
So for me is ready to be merged, what do you think @puellanivis?